corehq/apps/hqwebapp/static/hqwebapp/js/alert_user.js

```diff
@@ -25,7 +25,7 @@ function (
         var self = {
             "message": ko.observable(message),
             "alert_class": ko.observable(
-                "alert fade in message-alert"
+                "alert fadeIn message-alert"
```

Comment:
Do you know where `fadeIn` comes from? I didn't think bootstrap ever used camel case for class names, and we do use `fade in` elsewhere.

Reply:
It's defined in utils.less (https://github.com/dimagi/commcare-hq/blob/9757db6d9ed849e508b4c58fb3a190a687d7965d/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less#L28)

I see that `fade in` is the bootstrap classes. The transition time is 0.15s so it's just very fast which is why I thought it wasn't working. I'll revert to that.